LMChatCore_iOS/LMChatCore_iOS/Source/Module/MessageList/LMChatMessageListViewController.swift

```diff
@@ -262,6 +263,16 @@ open class LMChatMessageListViewController: LMViewController {
         self.present(alert, animated: true, completion: nil)
     }
     
+    @objc
+    open func navigateToSearchConversationScreen(){
+        do{
+            let searchListVC = try LMChatSearchListViewModel.createModule(searchOnlyConversations: true)
+            
```

Comment:
```suggestion
            let searchListViewController = try LMChatSearchListViewModel.createModule(searchOnlyConversations: true)
```

Use proper variable names